in_toto/examples_test.go

```diff
@@ -33,11 +33,11 @@ func ExampleInTotoVerify() {
 
 	// Perform in-toto software supply chain verification, using the provided
 	// test data.
-	var layoutMb Metablock
-	if err := layoutMb.Load(LayoutPath); err != nil {
+	layoutMb, err := LoadMetadata(LayoutPath)
+	if err != nil {
 		fmt.Printf("Unable to load layout metadata: %s", err)
 	}
-	if err := validateLayout(layoutMb.Signed.(Layout)); err != nil {
+	if err := validateLayout(layoutMb.GetPayload().(Layout)); err != nil {
```

Comment:
type assertion check to not cause a panic.

Reply:
So once again we're feeding in the data here, demo.layout.

I added a check in InTotoVerify to ensure it's a Layout more generally.